### PR TITLE
Feature/pruning

### PR DESCRIPTION
**Pull request by dioptre**
Added pruning functionality

Depends on https://github.com/ichliebedich/cql/pull/343

### CODE REVIEW

File: .setup/schema.3.cql [hunk 1]
```diff
@@ -56,6 +56,10 @@ create type payment (
   subtotal double, --what should have been paid (inc-tax)
   total double, --what should have been paid for the whole invoice (inc-tax)
   payment double, --payment = actual amount customer paid
+  currency text, --Ex EUR, USD
+  country text, --iso2 ex. US, DE, AU
+  rcode text, --State ex. CA, BW
+  region text, --State ex. California
```
Comment (cyberjunk) [inline .setup/schema.3.cql:62]: Are these missing in https://github.com/ichliebedich/cql/pull/343 ?
Should be created like the new `updated` columns below?

File: README.md [hunk 2]
```diff
@@ -140,7 +140,8 @@ Since GDPR, honest reporting about user telemetry is required. The default track
 * COOKIE_SESS (sess,sid): The session id. Each time you visit/use the site its approximately broken into session ids.
 * COOKIE_JWT (jwt): The encrypted token of your user. This may optionally include your user id (uid) if logged in.
 
-
+### Pruning Records
+* Run  ```./tracker --prune config.json``` to run privacy pruning.
```
Comment (cyberjunk) [inline README.md:144]: Please clarify: This is something we will run manually every once in a while to trigger the pruning or is it the new regular startup command for the tracker processes inside the dockers?

File: config.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "TempDirectory": "./tmp/",
     "UseGeoIP": true,
     "UseRegionDescriptions": true,
+    "UseRemoveIP": false,
```
Comment (cyberjunk) [inline config.json:17]: Please apply the same/appropriate changes to `config.homodea.json` (it's the file that we use).
Or let me know if I can just copy+paste (especially regarding the 'Prune' section)

Comment (dioptre) [reply config.json:17]: This is false by default, so left it out initially, it's relevant now so just including it here. Not important for our stuff yet.

File: README.md (hunk 2 above)
Comment (dioptre) [reply README.md:144]: This is a once in a while (crontab I'm envisioning)

File: .setup/schema.3.cql (hunk 1 above)
Comment (dioptre) [reply .setup/schema.3.cql:62]: I don't think so but we can look. I think it might be in another PR if not but I'll check.

Comment (dioptre) [reply .setup/schema.3.cql:62]: Yes I thought so, these changes were made 2 months ago in this commit https://github.com/ichliebedich/cql/pull/334/files

**Comment by dioptre** [comment]
@cyberjunk please merge this next days

File: .setup/schema.3.cql (hunk 1 above)
Comment (cyberjunk) [reply .setup/schema.3.cql:62]: @dioptre @IvanPetkow 
I have updated https://github.com/ichliebedich/cql/pull/343 and it's ready for merge.
We want to merge this here, too, but we haven't and we don't want to merge https://github.com/ichliebedich/cql/pull/334 right now. 
But mentioned fields are in there...
Please double check https://github.com/ichliebedich/cql/pull/334 for any dependency required for this task here, too and if necessary, please add them (e.g. fields listed above) to https://github.com/ichliebedich/cql/pull/343

**Comment by dioptre** [comment]
As already mentioned this work is updated in the ichliebedich cql
repository and is not relevant here.

**Review by dioptre — COMMENTED** [review]
I don't think supervisor should be used with the tracker.

**Pull request merged**